### DISCUSSION TITLE
Invalid db.major_version: 15

### DISCUSSION THREAD

**Discussion by ImadBoumzaoued** [supabase/supabase#11733] [Questions] [answered]
Trying to use the github action shown in this page but I keep getting this error

![image](https://user-images.githubusercontent.com/1586936/213159917-5e2f9945-5008-4eb6-bdf2-5e793493520f.png)

running `server_version` gives `15.1` in the server

any idea how to solve this?

**Comment by GaryAustin1** [comment, marked as answer]
Probably need to upgrade the CLI...
https://github.com/supabase/cli/issues/756

**Reply by ImadBoumzaoued** [reply]
That solved it, thanks :)
